models/resnet.py

```diff
@@ -11,7 +25,9 @@
 from model import Model
 from .download import get_weights_path
 
-__all__ = ['ResNet', 'resnet50', 'resnet101', 'resnet152']
+__all__ = [
+    'ResNet', 'resnet18', 'resnet34', 'resnet50', 'resnet101', 'resnet152'
```

Comment:
resnet18, resnet34, ... -> ResNet18, ResNet34, ...

Reply:
It is a function, so i think lower case will be good?